cmd/run/run.go

```diff
@@ -439,7 +447,7 @@ func (s *ServerContext) datastoreConfig(config *serverconfig.Config) (storage.Op
 			return nil, nil, fmt.Errorf("initialize mysql datastore: %w", err)
 		}
 	case "postgres":
-		datastore, err = postgres.New(config.Datastore.URI, dsCfg)
+		datastore, err = postgres.New(config.Datastore.URI, config.Datastore.SecondaryURI, dsCfg)
```

Comment:
would it make sense to just include the primary and secondary URIs in dsCfg? That would mean we can stick with the same `New(sqlcommon.Config)` signature for all the database adapters.